static/redfish/v1/JsonSchemas/OemAcf/index.json

```diff
@@ -0,0 +1,65 @@
+    "copyright": "Copyright 2014-2019 DMTF. For the full DMTF copyright policy, see http://www.dmtf.org/about/policies/copyright",
+    "definitions": {
+        "ACF": {
+            "additionalProperties": false,
+            "description": "OEM Extension for ACF",
+            "longDescription": "OEM Extension for ACF providing info about ACF Certificate generation.",
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "null",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {
+                "Model": {
+                    "description": "The product name for this system, without the manufacturer name.",
+                    "longDescription": "This property shall describe how the manufacturer refers to this system.  Typically, this value is the product name for this system without the manufacturer name.",
+                    "readonly": true,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+                "SerialNumber": {
+                    "description": "The serial number for this system.",
+                    "longDescription": "This property shall contain the serial number for the system.",
+                    "readonly": true,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+                "DateTime": {
+                    "description": "The current date and time with UTC offset of the log service.",
+                    "format": "date-time",
+                    "longDescription": "This property shall contain the current date and time with UTC offset of the log service.",
+                    "readonly": false,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+                "DateTimeLocalOffset": {
+                    "description": "The time offset from UTC that the DateTime property is in `+HH:MM` format.",
+                    "longDescription": "This property shall contain the offset from UTC time that the DateTime property contains.  If both DateTime and DateTimeLocalOffset are provided in modification requests, services shall apply DateTimeLocalOffset after DateTime is applied.",
+                    "pattern": "^([-+][0-1][0-9]:[0-5][0-9])$",
+                    "readonly": false,
+                    "type": [
+                        "string",
+                        "null"
+                    ]
+                },
+            },
+            "type": "object"
+        }
+    },
+    "owningEntity": "OpenBMC",
```

Comment:
Since you went OpenBMC here, let's go ["Oem"]["OpenBMC"]["DateTime"] , etc. 
If instead you go  ["Oem"]["IBM"]["DateTime"] let's change this to IBM

Reply:
Done.